package.json

```diff
@@ -1829,6 +1829,37 @@
                     "separator": "\\"
                 }
             }
+        ],
+        "walkthroughs": [
+            {
+                "id": "getStarted",
+                "title": "%AWS.walkthrough.gettingStarted.title%",
+                "description": "%AWS.walkthrough.gettingStarted.description%",
+                "when": "!config.aws.profile",
```

Comment:
Does this mean that this will always show up if a user has no AWS profile? Our functionality doesn't depend on AWS credentials (e.g. a user who only uses the toolkit for SAM debugging, and deploys via CI), so this can get annoying if that's the case.

Reply:
The walkthrough is not super intrusive. It just shows up on the start screen (no prompts/notifications/etc.)

Reply:
Hmm, interesting. I don't have a sense for how common those cases are, or whether those users would have a desire to graduate to the 'full' toolkit experience to which this walkthrough would be helpful. But to Jaden's point the walkthrough is not prominent and can be hidden. This when clause just controls the conditions in which the walkthrough would originally be surfaced through the welcome screen. I am open to suggestions on a better hook but I also don't want the conditions to be too narrow, i.e.: only on the first 'run' of the extension via a context setting, since a user may go months without using the Toolkit, end up with now invalid credentials, and has since forgotten the setup procedure.